chia-blockchain/defaults/main.yml

```diff
@@ -260,6 +260,7 @@ chia_add_healthcheck_service: true
 chia_healthcheck_port: 9950
 add_monit_config: true
 monit_start_now: true
+chia_alerts_receiver_token: "{{ lookup('env', 'CHIA_ALERTS_RECEIVER_TOKEN') }}"
```

Comment:
For a defaults file, this should just be an empty string. The env lookup should be in the playbook/vars that actually call this role